### ISSUE TITLE
Is it possible to automatically expand columns?

### ISSUE THREAD

**Issue by simkessy** [pnp/pnpjs#208]
Hi, if I'm quering a list and there are columns that are expandable, is there a way to automatically get the expanded data when I build my request without having to be explicit about it?

**Comment by patrick-rodgers** [comment]
No, so far as I know you need to [specify the fields to expand](https://pnp.github.io/pnpjs/sp/items.html#retrieving-lookup-fields).

**Comment by simkessy** [comment]
Okay thanks!

**Comment by patrick-rodgers** [comment]
Any comment on why this was reopened?
